app/utils/DefaultGetResponse.scala

```diff
@@ -43,35 +43,7 @@ object DefaultGetResponse {
 
   def defaultGetEvent1832(): JsValue = Json.parse("{}")
 
-  def defaultGetEvent1833(pstr: String, version: String, startDate: String): JsValue = Json.parse(
-    s"""
-       | {
-       |   "processingDate": "2023-12-15T12:30:46Z",
-       |   "schemeDetails": {
-       |     "pSTR": "$pstr",
-       |     "schemeName": "Abc Ltd"
-       |   },
-       |   "er1Details": {
-       |     "reportStartDate": "$startDate",
-       |     "reportEndDate": "2022-04-05",
-       |     "reportVersionNumber": "$version",
-       |     "reportSubmittedDateAndTime": "2023-12-13T12:12:12Z"
-       |   },
-       |   "schemeMasterTrustDetails": {
-       |     "startDate": "2021-06-08"
-       |   },
-       |   "erDeclarationDetails": {
-       |     "submittedBy": "PSP",
-       |     "submittedID": "20000001",
-       |     "submittedName": "ABCDEFGHIJKLMNOPQRSTUV",
-       |     "pspDeclaration": {
-       |     "authorisedPSAID": "A4045157",
-       |     "pspDeclaration1": "Selected",
-       |     "pspDeclaration2": "Selected"
-       |   }
-       |   }
-       |
-       |}""".stripMargin)
+  def defaultGetEvent1833(): JsValue = Json.parse("{}")
```

Comment:
What's the impact of this change?

Reply:
This default response was actually incorrect for the API it was stubbing. If we try to GET for 1833 and there is no data for that PSTR in `example-pstr.json`, then we return `{}`. There is now a correct stub under `conf/resources/data/api1833/87219363YN.json` which is the same PSTR used in the Postman tests.